drivers/storage/portworx/component/portworx_basic.go

```diff
@@ -379,6 +379,11 @@ func (c *portworxBasic) createClusterRole() error {
 					Resources: []string{"*"},
 					Verbs:     []string{"*"},
 				},
+				{
+					APIGroups: []string{"portworx.io"},
+					Resources: []string{"portworxdiags", "portworxdiags/status"},
```

Comment:
Do we need to mention portworxdiags/status explicitly here if we have portworxdiags in the resource list?

Reply:
Yes, I believe that was a change in how Kubernetes handles object RBAC, it's to prevent non-controllers from accidentally updating the status of controlled objects